### DISCUSSION TITLE
Stand Up Tuesday May 30th

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#86] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
-  Worked on Architecture Document

**What are your priorities for the next 24 hours?**:
- Keep working on Arch doc
- Penetration Testing Doc
- Web Doc

**Blockers**:
-  Lack of will

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
-  Not much tbh

**What are your priorities for the next 24 hours?**:
- Keep working on Arch doc
- Penetration Testing Doc
- Web Doc

**Blockers**:
-  Im sick

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on how the requests for graphs are goin to be made.
- Solved SSG issues

**What are your priorities for the next 24 hours?**:
- Finished the interface/middleware or idk that is going to be making the requests

**Blockers**:
- None

**Shoutouts**:
- @Jacobgtz024 bc he still pumping out graphs

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Worked on algorithm to group IBM certifications (585)
- Planned the radar graph for every certification in IBM

**What are your priorities for the next 24 hours?**:
- Finish both radar graphs

**Blockers**:
- None

**Shoutouts**:
- @CarlosCRG19 for grinding in the gym
